source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+
+Until recently app wrapping has been the common approach. It does not come without risks and challenges to scalability though. 
+
+For example, most modern applications follow continuous development. Each time an application changes it will need to go through the process described above. 
+
+There is also functionality and compatibility risk. This is a known issue for the Mattermost application and app wrapping tools.
+
+This incompatibility is not an issue with the Mattermost application. It results from the proprietary nature of the provider's tools. To make matters worse, there is no course of action to address compatibility issues.
+
+In the end, this sets app wrapping in a negative light. This is the reason VMWare and other EMM providers came together to create a standard. AppConfig is a modern, efficient, and scalable approach to enterprise mobile management. In that regard, the Mattermost application benefits from being built around this standard.
+
+******************
+Mobile VPN Options
+******************
+
+A VPN allows a device outside a firewall to access content inside the firewall as if it were on the same network. Most enterprise teams are familiar with VPNs (Virtual Private Networks). We won't go into detail here.
+
+There are VPN options which depend on the requirements of your organization. You should also consider the demands/needs of your user. Regardless, this can impact your approach to mobile deployment.
+
+For the Mattermost mobile application, we’ll discuss two options: a device VPN or per-app VPN.
+
+    **Note:** We suggest `following our recommended steps <https://docs.mattermost.com/mobile/mobile-appstore-install.html>`_ to secure your deployment.
+
+**Device VPN**
+    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issue for personal applications. Of course even for devices provided by your business, this tends not to be the modern approach.
+
+**Per-app VPN**
+    In contrast, the more common approach is to use a per-app VPN. This provides a connection to the VPN when needed (on-demand). For example, when using a particular app.
+
+Regardless of the commonness of either option above, Mattermost provides support for both. Because Mattermost supports the AppConfig standard, both options are compatible with EMM providers.
+
+    **Note:** Will you be connecting via a corporate proxy server? If so, `review our FAQ <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`_ covering architecture, troubleshooting, and best practices.
+
+----
```

Comment:
Something I hadn't picked up on reading this in previous drafts was that we have a whole section on Mobile App VPN, but I find in my experience that is far less commonly used than reverse proxy back to the MM server for remote mobile and webapp connections. We haven't touched on the requirements for an inbound reverse proxy, we have naturally focused more on the outbound as that has focus at present. For an inbound reverse proxy if it doesn't support websockets then the app isn't going to work. Feels a gap. Not sure if you want to include guidance now or later but it would be good to get others input on this as a subject.

Reply:
VPN was what we provided as part of our internal training. @johnthompson365 do you think this is critical to get in? If so, could you work with Sebastian to add the section and confirm technical accuracy with the dev team? It seems like we’d just need to add another section for **Revers Proxy** and then a few sentences  to describe it at a high level.

Reply:
Thought about this some more. To get this delivered I think this would be something we could work on and iterate delivery. Unless there is a strong feeling it can’t go out without it.

Reply:
5/5 To move this to a v2. we can create an item in our backlog to add content related to Reverse Proxy for this guide.